js/customize-snapshots.js

```diff
@@ -765,9 +765,11 @@
 
 			// Change update button to schedule.
 			if ( component.isFutureDate() ) {
-				save.html( component.data.i18n.scheduleButton );
+				save.text( component.data.i18n.scheduleButton );
+			} else if ( api.state( 'snapshot-exists' ).get() ) {
+				save.text( component.data.i18n.updateButton );
 			} else {
-				save.html( component.data.i18n.updateButton );
+				save.text( component.data.i18n.saveButton );
```

Comment:
We could move this logic to `addButton` but we also want to tie this logic on date input change so I kept it separate. But yes I see the advantage of keeping all button logic in `addButton`. 
Maybe we should create a event for schedule and bind this button logic in `addButton` with that event?